pom.xml

```diff
@@ -2651,6 +2660,25 @@
                                     <resource>opencensus/proto/trace/v1/trace_config.proto</resource>
                                 </resources>
                             </exception>
+                            <exception>
```

Comment:
This should be covered by the above exclusion and can be removed here

Reply:
done